pkg/plugin/view/vulnerabilities/report_view.go

```diff
@@ -95,15 +95,16 @@ func NewReport(workload kube.Object, vulnerabilityReportsDefined bool, reports [
 	return flexLayout
 }
 
-func createVulnerabilitiesTable(containerName string, report starboard.VulnerabilityReport) component.Component {
+func createVulnerabilitiesTable(containerName string, report starboard.VulnerabilityScanResult) component.Component {
+	imageRef := fmt.Sprintf("%s/%s:%s", report.Registry.Server, report.Artifact.Repository, report.Artifact.Tag)
 	table := component.NewTableWithRows(
-		fmt.Sprintf("Container %s", containerName), "There are no vulnerabilities!",
+		fmt.Sprintf("Container %s: %s", containerName, imageRef), "There are no vulnerabilities!",
```

Comment:
```suggestion
		fmt.Sprintf("Container %s:%s", containerName, imageRef), "There are no vulnerabilities!",
```

Would this change result in it looking more like (for example) `nginx:1.6`

Reply:
It's going to look like `Container nginx: docker.io/library/nginx:1.6`

In general this is `Container <container name>: <image ref>`